src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -54,9 +54,7 @@ export class DictionaryOverview extends Component {
     this.props.fetchDictionary(url);
     this.props.fetchVersions(versionUrl);
 
-    const conceptsUrl = `
-      /${ownerType}/${owner}/collections/${name}/concepts/?q=&limit=0&page=1&verbose=true
-    `;
```

Comment:
Is this supposed to be in this ticket?

Reply:
Yes, Because of the concepts were not being fetched due to malformed endpoint URL. And I can't search functionality was using this endpoint.